modeling-cmds/src/ok_response.rs

```diff
@@ -419,6 +419,14 @@ define_ok_modeling_cmd_response_enum! {
             /// The number of objects in the scene.
             pub num_objects: u32,
         }
+
+        /// The response from the `MakeOffsetPath` command.
+        #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, ModelingCmdOutput)]
+        pub struct MakeOffsetPath {
+            /// The UUIDs of the paths that were created.
```

Comment:
This is only if the path splits into "islands" right? I'd say so, otherwise it's confusing whether this is the same as the new ID the client/request has chosen.

Reply:
Yeah I wasn't sure how to handle it if multiple islands are created. Do you think this should be an `Option` then and that it the signal that the path has split?